run.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2014,2015,2016 Docker, Inc.
```

Comment:
Good catch, we should remove this, it is coming from the copy/paste from runc I guess.